charts/apps/kubeflow-pipelines/rds-s3-static/values.yaml

```diff
@@ -1,6 +1,6 @@
 rds:
   dbHost: 
-  mlmdDb: metadb
+  mlmdDb: 
```

Comment:
revert?

Reply:
it should be what ever specified in here: https://github.com/awslabs/kubeflow-manifests/blob/main/tools/helmify/template/kubeflow-pipelines/values/rds-s3-static/values.yaml I think it was manually changed during testing from previous PR and did not change back. What I have in this PR should be right.